plugins/source/aws/docs/tables/aws_cloudwatchlogs_log_group_subscription_filters.md

```diff
@@ -0,0 +1,27 @@
+# Table: aws_cloudwatchlogs_log_group_subscription_filters
+
+https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_SubscriptionFilter.html
+
+The composite primary key for this table is (**account_id**, **region**, **creation_time**, **filter_name**).
```

Comment:
No log group in the PKs? maybe add `log_group_arn` and remove account_id/region? (can't say anything about the creation time being in the PKs...)